src/realms/twitter/router.ts

```
@@ -30,28 +31,25 @@ export const getBaseRedirectUrl = (c: Context) => {
-      faviconUrl = 'https://abs.twimg.com/favicons/twitter.2.ico';
-    }
-    const response = await fetch(faviconUrl);
+    const response = await fetch(branding.favicon);
     const body = await response.arrayBuffer();
     return c.body(body, response.status as ContentfulStatusCode, {
       'Content-Type': response.headers.get('Content-Type') || 'image/x-icon',
-      'Content-Length': response.headers.get('Content-Length') || body.byteLength.toString(),
-    },);
+      'Content-Length': response.headers.get('Content-Length') || body.byteLength.toString()
+    });
   } catch (e) {
```

ESLint

[error] 'e' is defined but never used.

(@typescript-eslint/no-unused-vars)